org.btchd.btchd-qt.desktop

```diff
@@ -0,0 +1,14 @@
+[Desktop Entry]
```

Comment:
FYI ideally you should maintain it upstream.

Reply:
I asked but the owner is opposed to it for whatever reason.